openupgradelib/openupgrade.py

```diff
@@ -582,6 +583,65 @@ def rename_columns(cr, column_spec):
                 )
 
 
+def _rename_field_on_dashboard(env, model, old_field, new_field,
+                               dashboard_view_data):
+    for r in dashboard_view_data:
```

Comment:
Do the search here and only if the view contains `old_field`.

Reply:
Thanks for the review and suggestion, I'm quite busy at the moment so i will comeback with this ASAP. In the meantime, feel free to add some modification to this PR, i don't mind.

Kind regard,

Reply:
@pedrobaeza Forgive me, but is all you are suggesting is (in my simplistic ways)

```
if old_field not in r.arch:
    continue
```

Reply:
More or less, for improving drastically the performance. The previous search shouldn't contain non matched results.

Reply:
Done i use `ilike` search for `ir.ui.view.custom` that contain the `old_field` , is that ok? , also i rename the method and handle some of your comment, please review if you have time